components/Main/Pages/RestPage.js

```diff
@@ -19,19 +19,37 @@ function Timer({interval}) {
 }
 
 const RestPage = (props) => {
+  const [task, setTask] = useState({
+    id: "1"
+  });
   return (
     <View style={styles.container}>
-      <LinearGradient
-        colors={['#C8E0E1', 'white']}
-        style={styles.linearGradient}>
-        <Timer interval={props.Time} />
-        <Text>We are resting</Text>
-        <Button title="Back to Work" onPress={props.Resume} />
-        <Button
-          title="End Session"
-          onPress={() => props.navigation.navigate('Status Report', {time: props.Time})}
-        />
-      </LinearGradient>
+      <Text style={styles.text}>Work Time</Text>
```

Comment:
The time over here should be rest time

Reply:
Also the font for task looks different then the one katie have can you change that?